src/main.js

```diff
@@ -130,9 +109,7 @@ let load = base.loader({
           publicBucket:   ctx.publicArtifactBucket,
           privateBucket:  ctx.privateArtifactBucket
         },
-        drain:            ctx.influx,
-        component:        ctx.cfg.app.statsComponent,
-        process:          ctx.process
+        monitor: ctx.monitor.prefix('artifact'),
```

Comment:
Maybe a pattern like `table.<tablename>` (ie. `table.artifacts`) is better?

Just because I can totally imagine us adding custom statistics about artifacts... Like size, etc... that isn't related to the azure table at all..